form/form.php

```diff
@@ -106,7 +105,7 @@
 			$output .= '</div>';
 		} // End foreach().
 	} // End if().
-	echo $output;
+	echo wp_kses_post( $output );
```

Comment:
@pdclark seems like this is affecting the modal in the classic editor widget, the inputs not being available anymore.

Reply:
Thanks! Fixed to more broad `wp_kses` with whitelist for form elements and `feedzy-data` attributes in latest push:
https://github.com/Codeinwp/feedzy-rss-feeds/pull/521

Code:
https://github.com/Codeinwp/feedzy-rss-feeds/pull/521/commits/34eea2d181f6cc2c6d8e2535eacaf18188cc2543